django/contrib/admin/checks.py

```diff
@@ -70,10 +83,15 @@ def check_dependencies(**kwargs):
             id='admin.E403',
         ))
     else:
-        if ('django.contrib.auth.context_processors.auth'
-                not in django_templates_instance.context_processors and
-                'django.contrib.auth.backends.ModelBackend'
-                in settings.AUTHENTICATION_BACKENDS):
+        auth_ctx_processor_installed = (
+            'django.contrib.auth.context_processors.auth'
+            in django_templates_instance.context_processors
+        )
+        auth_model_backend_installed = _module_exists(
+            'django.contrib.auth.backends.ModelBackend',
+            settings.AUTHENTICATION_BACKENDS
+        )
+        if (auth_model_backend_installed and not auth_ctx_processor_installed):
```

Comment:
I understand the motive behind the reformatting of the conditionals but I'd stick to the previous one to reduce the patch noise ratio.

Reply:
Submitted a new patch-set now. It's tricky as the lines become so damn long when the module paths take much of it – but I simplified it a bit now :heart_eyes: